src/PublicLab.Editor.js

```diff
@@ -81,6 +82,22 @@ PL.Editor = Class.extend({
       _editor.data[attrname] = options.data[attrname];
     }
 
+    // shows less message in editors footer on small screens, 
```

Comment:
Comments start with title case (Title case). Also, a better comment here would be "Shortens agreement message on mobile screens/devices".

Reply:
I feel it doesn't just shorten the agreement message. The shortening is done somewhere else; it shows this "By publishing, you agree to open source your work" on mobile, instead of this "By publishing, you agree to open source your work so that others may use it".  Won't this comment make someone new think the shortening was done here?

I think if the comment should be added, it should mention that the code hides and expands the agreement message.

Reply:
I still feel that the JS should be removed altogether.
This responsiveness can be done with HTML + CSS, adding JS just creates a mess.

Reply:
To achieve the same result, I'll still need to add a click event handler. And not creating an additional style on another class is how I have simplified things. 

Maybe you have not tried making or editing a post on publiclab.org/post on a mobile phone.  If you have, you'd probably get a better sense of what I'm trying to fix here.  

My point is, you can't avoid adding an event handler if we want to reduce how much space the footer occupies, while still having the user get read the agreement message.

Reply:
I was referring to the line 88 where you cut out some content via JS.

Reply:
Okay.